docs/intro/reusable-apps.txt

```diff
@@ -61,9 +61,9 @@ After the previous tutorials, our project should look like this:
         manage.py
         mysite/
             __init__.py
+            asgi.py
             settings.py
             urls.py
-            asgi.py
             wsgi.py
```

Comment:
Added out of order in 3930ec1bf275d17f1c36cd38b38114177b1d1565

Reply:
Is the order really important here? I wanted to put it next to the `wsgi.py`.

Reply:
All file listings in the different tutorial steps and elsewhere in the docs are alphabetical. I see no reason to special-case `asgi.py`. I will extract these changes into a separate PR.

Reply:
See #17680.